apps/x86/multi/app_settings.cmake

```
@@ -0,0 +1,18 @@
+#
+# Copyright 2022, UNSW (ABN 57 195 873 179)
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+cmake_minimum_required(VERSION 3.8.2)
+
+# Define kernel config options
+set(KernelSel4Arch x86_64 CACHE STRING "" FORCE)
```

CMake does not require quotes, but it's a bit nicer to use them for strings.
```suggestion
set(KernelSel4Arch "x86_64" CACHE STRING "" FORCE)
```